ORBIT/manager.py

```diff
@@ -1015,6 +1200,33 @@ def turbine_capex_per_kw(self):
         _capex = self.config.get("turbine_capex", None)
         return _capex
 
+    @property
+    def total_capex(self):
+        """
+        Returns total BOS CAPEX including commissioning and decommissioning.
+        """
+
+        return (
+            self.bos_capex
```

Comment:
where is self.bos_capex being calculated?  i can't seem to find it.

Reply:
https://github.com/WISDEM/ORBIT/blob/4f476f0d062e395dc26f82da1ae58c830d20e0a1/ORBIT/manager.py#L1087-L1093

It probably wasn't showing up because it didn't get edited in this PR.